pipeline/cloud/schemas/runs.py

```diff
@@ -130,6 +131,11 @@ def from_object(cls, obj: t.Any):
             except (TypeError, OverflowError):
                 return cls.pkl
             return cls.array
+        # Also check if class name == Stream
+        elif hasattr(obj, "iterable") or (
+            hasattr(obj, "__name__") and obj.__name__ == "Stream"
+        ):
```

Comment:
I'm not too familiar with this code but can we not just check if `isinstance(obj, Stream)`?

Reply:
I think I did this because your suggestion would cause a circular dependency

Reply:
ah ok. Moving the `Stream` class into a separate module may help